backend/CircleForms/Models/Posts/Post.cs

```diff
@@ -19,6 +19,9 @@ public Post()
     [JsonProperty("author_id")]
     public string AuthorId { get; set; }
 
+    [JsonProperty("is_active")]
+    public bool IsActive { get; set; }
```

Comment:
Might as well add `DateTime ActiveUntil` and related filtration checks (see #56)

Reply:
later